src/main/java/racingcar/controller/InputController.java

```diff
@@ -0,0 +1,17 @@
+package racingcar.controller;
+
+import camp.nextstep.edu.missionutils.Console;
+import racingcar.domain.InputContainer;
+
+public class InputController {
+
+    public InputContainer readConsole() {
+        System.out.println("경주할 자동차 이름을 입력하세요.(이름은 쉼표(,) 기준으로 구분)");
+        String inputCarNames = Console.readLine();
+        System.out.println("시도할 횟수는 몇 회인가요?");
+        int attempts = Integer.parseInt(Console.readLine());
```

Comment:
사소하지만 _공백도 코딩 컨벤션이다_ 라는 공통 피드백처럼 공백 넣어주시면 두 입력이 더 잘 구분될 것 같습니다👀

```suggestion
        System.out.println("경주할 자동차 이름을 입력하세요.(이름은 쉼표(,) 기준으로 구분)");
        String inputCarNames = Console.readLine();

        System.out.println("시도할 횟수는 몇 회인가요?");
        int attempts = Integer.parseInt(Console.readLine());
```